src/app/time-select/time-select.component.less

```diff
@@ -0,0 +1,55 @@
+@import '../common/styles/variables';
+
+& {
+  .controls {
+    text-align: center;
+    padding: 10px 0;
+    line-height: normal;
+    margin: 0;
+  }
+  .control {
+    vertical-align: middle;
+    display: inline-block;
+    font-size: inherit;
+    margin: 0 auto;
+    width: 35px;
+    letter-spacing: 1.3px;
+  }
+  .control-up, .control-down {
+    position: relative;
+    display: block;
+    margin: 3px auto;
+    font-size: 18px;
+    font-weight: bold;
+    cursor: pointer;
+    &:before {
+      display: inline-block;
+      position: relative;
+    }
+  }
+  .control-up:before {
+    content: '︿';
```

Comment:
We shouldn't use this type of characters as they are probably not supported on linux or other OSs. Please use pure CSS.